camera-core/src/main/java/com/stripe/android/camera/scanui/ScanFlow.kt

```diff
@@ -1,4 +1,4 @@
-package com.stripe.android.stripecardscan.scanui
+package com.stripe.android.camera.scanui
```

Comment:
should we make the scan flow stream generic? something like `dataStream: Flow<DataType>` ?

Reply:
We'll use the same `CameraPreviewImage<Bitmap>` type in identity as they come from the same `CameraAdapter`, but doesn't hurt to make them more generic :) updated